masterlist.yaml

```
@@ -9291,6 +9291,25 @@ plugins:
         itm: 88
         udr: 512
 
+  - name: 'IC Circular Canal.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/49868/'
+        name: 'Imperial City Canal Overhaul'
+    inc:
+      - 'Better Imperial City.esp'
+      - 'Better Cities IC Talos Plaza.esp'
+      - 'Better Cities IC Elven Gardens.esp'
+      - 'Better Cities IC Market.esp'
+      - 'Better Cities IC Arena.esp'
+      - 'Better Cities IC Arboretum.esp'
+      - 'Better Cities IC Temple.esp'
```

These should be sorted alphabetically.

```suggestion
      - 'Better Cities IC Arboretum.esp'
      - 'Better Cities IC Arena.esp'
      - 'Better Cities IC Elven Gardens.esp'
      - 'Better Cities IC Market.esp'
      - 'Better Cities IC Talos Plaza.esp'
      - 'Better Cities IC Temple.esp'
      - 'Better Imperial City.esp'
```